packages/react/src/Icon/Icon.tsx

```diff
@@ -9,27 +9,28 @@ import clsx from 'clsx'
 import { forwardRef } from 'react'
 import type { ForwardedRef, HTMLAttributes, ReactNode } from 'react'
 
+export const iconSizes = ['body-text-small', 'body-text-large', 'heading-3', 'heading-4', 'heading-5', 'heading-6']
```

Comment:
Can we get away with removing `body-text-` from these first two options? `<Icon size="small">` is much more intuitive and people will understand that it refers to the small body text, right?